arbos/util/tracing.go

```diff
@@ -47,6 +48,26 @@ func NewTracingInfo(evm *vm.EVM, from, to common.Address, scenario TracingScenar
 	}
 }
 
+func (info *TracingInfo) RecordEmitLog(topics []common.Hash, data []byte) {
+	size := uint64(len(data))
+	var args []uint256.Int
+	args = append(args, *uint256.NewInt(0))
+	args = append(args, *uint256.NewInt(size))
```

Comment:
nitpick: a comment indicating that those values are related to offset and length, as it is done in other parts of this file, can help code readers